src/traces/contour/plot.js

```diff
@@ -543,8 +544,8 @@ function locationCost(loc, textOpts, labelData, bounds) {
 }
 
 exports.addLabelData = function(loc, textOpts, labelData, labelClipPathData) {
-    var halfWidth = textOpts.width / 2;
-    var halfHeight = textOpts.height / 2;
+    var halfWidth = textOpts.width / 2 + LABEL_LINE_CLIP_PAD;
+    var halfHeight = textOpts.height / 2 + LABEL_LINE_CLIP_PAD;
```

Comment:
After viewing the baseline changes, I think we may not need to add vertical padding to the height.

Reply:
I agree, let's not add vertical padding. In fact if there was a way to make it even a tighter fit vertically than it currently is on master, I'd be all for it. Probably not though, assuming that comes from a bounding box... but the goal is to avoid clipping neighboring contour lines as much as possible, since that can cause confusion about which line is actually being labeled.